tools/accuracy_checker/configs/im2latex-0001.yml

```diff
@@ -0,0 +1,65 @@
+evaluations:
+  - name: im2latex-medium-0002
+    module: custom_evaluators.im2latex_evaluator.Im2latexEvaluator
+    module_config:
+      network_info:  
```

Comment:
some style errors detected in your config, could you please fix?
```
16:13:16    5:20      error    trailing spaces  (trailing-spaces)
16:13:16    39:1      error    trailing spaces  (trailing-spaces)
16:13:16    59:1      error    trailing spaces  (trailing-spaces)
16:13:16    63:1      error    trailing spaces  (trailing-spaces)
16:13:16    64:1      error    trailing spaces  (trailing-spaces)
16:13:16    65:47     error    no new line character at the end of file  (new-line-at-end-of-file)
```

Reply:
Fixed